integrated_channels/canvas/dev_docs/postman_collections/Canvas.postman_collection.json

```diff
@@ -0,0 +1,445 @@
+{
+	"info": {
+		"_postman_id": "1e52ea29-e08c-4247-879a-04bd5d82a713",
+		"name": "Canvas",
+		"description": "APIs for interacting with Canvas LMS",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET oauth code",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/login/oauth2/auth?client_id=10000000000002&response_type=code&state=edx-canvas-user-state&redirect_uri=https://example.com/oauth_complete",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"login",
+						"oauth2",
+						"auth"
+					],
+					"query": [
+						{
+							"key": "client_id",
+							"value": "{{client_id}}"
+						},
+						{
+							"key": "response_type",
+							"value": "code"
+						},
+						{
+							"key": "state",
+							"value": "edx-canvas-user-state"
+						},
+						{
+							"key": "redirect_uri",
+							"value": "https://example.com/oauth_complete"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET Oauth key",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "grant_type",
+							"value": "authorization_code",
+							"type": "text"
+						},
+						{
+							"key": "client_id",
+							"value": "{{client_id}}",
+							"type": "text"
+						},
+						{
+							"key": "client_secret",
+							"value": "{{client_secret}}",
+							"type": "text"
+						},
+						{
+							"key": "code",
+							"value": "{{code}}",
+							"type": "text"
+						}
+					]
+				},
+				"url": {
+					"raw": "{{canvas_url}}/login/oauth2/token",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"login",
+						"oauth2",
+						"token"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "list courses",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/courses",
```

Comment:
so according to the doc this endpoint gets courses for the "current user". I'm not sure how well this will work if the given user is not actually enrolled in any of these courses. I wonder if we should go by account id here, ie `{{canvas_url}}/api/v1/accounts/:account_id/courses/` but what has been your experience with courses you aren't specifically enrolled in?